### PR TITLE
Adds a spectroscopic monocle as a Bartender job reward

### PR DESCRIPTION
**Pull request by Carbadox**
## About the PR 
Adds a spectroscopic monocle as a Bartender job reward

## Why's this needed? 
more job rewards = epic
having an item that has the same properties as spectroscopic goggles with only a smaller sprite shouldnt affect any balance so its a neat thing to have I think

## Changelog

```
(u)Carbadox
(+)Added a new Bartender job reward, the spectroscopic monocle.
```

### CODE REVIEW

**Comment by Emcee-Gore** [comment]
Yeah - good change.

**Comment by Azrun** [comment]
One one hand... There a reason to not have it consume an existing `/obj/item/clothing/glasses/spectro`, which Bartenders should have immediate access on roundstart join? Otherwise it does seem like a mechanical advantage to peeps with level 10.

On the other hand... the flavor is sweet! 👍

**Comment by flrsh** [comment]
Nice flavor. Agreed with what Azrun said.

**Comment by Carbadox** [comment]
I'll make it require the glasses but I'll also take a quick sweep through all the maps to make sure there's always an easy available pair the barman can pick up

File: code/WorkInProgress/JobXPRewards.dm [hunk 1]
```diff
@@ -522,6 +522,27 @@ mob/verb/checkrewards()
...
+/datum/jobXpReward/bartender/spectromonocle
+	name = "Spectroscopic Monocle"
+	desc = "Now you can look dapper and know which drinks you poisoned at the same time"
+	required_levels = list("Bartender"=10)
+	icon_state = "?"
+	claimable = 1
+	var/path_to_spawn = /obj/item/clothing/glasses/spectro/monocle
+
+	activate(var/client/C)
+		var/obj/item/clothing/glasses/spectro/glasses = locate(/obj/item/clothing/glasses/spectro) in C.mob.contents
```
Comment (flrsh) [inline code/WorkInProgress/JobXPRewards.dm:534]: You may wanna do `var/glasses = C.mob.find_in_hand(/obj/item/clothing/glasses/spectro)`.

**Review by flrsh — CHANGES_REQUESTED** [review]

File: code/WorkInProgress/JobXPRewards.dm (hunk 1 above)
Comment (Carbadox) [reply code/WorkInProgress/JobXPRewards.dm]: done

**Review by flrsh — APPROVED** [review]

**Pull request merged**